util/dvsim/dvsim.py

```diff
@@ -601,10 +604,15 @@ def main():
 
     args.scratch_root = resolve_scratch_root(args.scratch_root)
     args.branch = resolve_branch(args.branch)
-    args.cfg = os.path.abspath(args.cfg)
-    proj_root = resolve_proj_root(args)
+    proj_root_src, proj_root_dest = resolve_proj_root(args)
+    proj_root = proj_root_dest
```

Comment:
I think the extra var is not required. Either keep `proj_root` or `proj_root_dest`, and update refs everywhere else.

Reply:
Sounds good! Thanks Sri, will update that!